Change docs according self-signed certificate flag automation (#1331)
codenvy/che-docs: src/main/pages/che-7/installation-guide/assembly_installing-che-in-tls-mode-with-self-signed-certificates.adoc

:context: installing-{prod-id-short}-in-tls-mode-with-self-signed-certificates

> WARNING: Self-signed certificates is usually used in development and evaluation enviromnents. If you want to use them in production you should understand the consequences.

Reviewer: ```suggestion
WARNING: Self-signed certificates are usually used in development and evaluation environments. Use in production environments is not recommended.
```

> WARNING: Self-signed certificates is usually used in development and evaluation enviromnents. If you want to use them in production you should understand the consequences.

Reviewer 2: enviromnents -> environments
